### PR TITLE
Updates logging to be more consistent

### PR DESCRIPTION
**Pull request by graybush**
The way the logging was initialized varied from file to file. This just
updates the init to be the same everywhere. Also updates some LOG.warn
to be LOG.info.

### CODE REVIEW

**Comment by oddstr13** [comment]
Diff is a bit large to review on mobile, but it looks like there are more changes here than described.

File: resources/lib/client.py [hunk 1]
```diff
@@ -82,12 +83,7 @@ def get_device_id(reset=False):
     if client_id:
         return client_id
 
-    directory = xbmc.translatePath('special://profile/addon_data/plugin.video.jellyfin/').decode('utf-8')
-
-    if not xbmcvfs.exists(directory):
```
Comment (cvium) [inline resources/lib/client.py]: Why was this removed?

File: resources/lib/database/__init__.py [hunk 2]
```diff
@@ -211,50 +212,37 @@ def reset():
...
-
-    if dialog("yesno", heading="{jellyfin}", line1=_(33087)):
-
-        xbmcvfs.delete(os.path.join(addon_data, "settings.xml"))
-        xbmcvfs.delete(os.path.join(addon_data, "data.json"))
-        LOG.info("[ reset settings ]")
-
-    if xbmcvfs.exists(os.path.join(addon_data, "sync.json")):
-        xbmcvfs.delete(os.path.join(addon_data, "sync.json"))
+    if window('jellyfin_sync.bool'):
```
Comment (cvium) [inline resources/lib/database/__init__.py]: What is this change for?

File: resources/lib/database/__init__.py [hunk 3]
```diff
@@ -346,49 +329,34 @@ def get_sync():
...
-        xbmcvfs.mkdirs(path)
-
     try:
-        with open(os.path.join(path, 'data.json')) as infile:
+        if os.path.exists(os.path.join(ADDON_DATA, 'data.json')):
+            pass
+        elif os.path.exists(os.path.join(ADDON_DATA, 'data.txt')):
+            os.rename(os.path.join(ADDON_DATA, 'data.txt'), os.path.join(ADDON_DATA, 'data.json'))
+        else:
+            open(os.path.join(ADDON_DATA, 'data.json'), 'a')
```
Comment (cvium) [inline resources/lib/database/__init__.py]: This doesn't seem right. What is it supposed to do?

File: service.py [hunk 4]
```diff
@@ -68,29 +70,32 @@ def run(self):
...
+    LOG.info("Delay startup by %s seconds.", DELAY)
+
+    if not xbmcvfs.exists(ADDON_DATA):
+        xbmcvfs.mkdirs(ADDON_DATA)
 
     while True:
 
-        if not settings('enableAddon.bool'):
-            LOG.warn("Jellyfin for Kodi is not enabled.")
+        if settings('enableAddon.bool'):
```
Comment (cvium) [inline service.py]: Why invert the if? It seems only the indentation level has been increased, so I don't think it was necessary

**Comment by graybush** [comment]
So this was just supposed to be just the logging stuff. I didn't realize
that since I had not created a branch additional changesets added to master
would automatically get added to this pull request. I will create a branch
for future changesets.

**Comment by oddstr13** [comment]
I've added a bunch of logging recently. Most of the exceptions aren't actually crashes, just over-eager logging of except blocks. Some have been nececary to track down underlying issues (catching Exception everywhere hides many bugs).

We're now printing a bunch of stack traces on caught exceptions, to make debuging things easier. As such, a stack trace may not equal a bug.
